keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -114,12 +117,16 @@ EnsureImagesExist step the kubelet will require authentication of present images
 Optimize to only force re-authentication for a pod container image when the
 `ImagePullSecrets` used to pull the container image has not already been authenticated.
 IOW if an image is pulled with authentication for a first pod, subsequent pods that have the same
-authentication information should not need to re-authenticate, unless the kubelet's `pullImageSecretRecheckPeriod` has passed.
+authentication information should not need to re-authenticate.
 
-Images already present at boot or loaded externally to the kubelet or successfully
-pulled through the kubelet with no `ImagePullSecrets`/authentication required will
+Images already present at boot or loaded externally to the kubelet will not require
+authentication unless configured otherwise.
+Images successfully pulled through the kubelet with no `ImagePullSecrets`/authentication required will
 not require authentication.
 
+The new behavior is designed in a way so that it replaces current behavior - it is going
+to be an on-by-default feature.
```

Comment:
nit: clarify that it will be on-by-default after it graduates, this isn't proposing direct-to-beta-and-enabled

```suggestion
to be an on-by-default feature once graduated.
```